src/js/mixin/svg.js

```
@@ -0,0 +1,121 @@
+    dimensions.forEach((val, i) => attr(el, props[i], toFloat(val) * this.ratio || null));
+}
+
+export function parseSVG(svg, icon) {
+    if (icon && includes(svg, '<symbol')) {
+        svg = parseSymbols(svg)[icon] || svg;
+    }
+
+    return toNodes(fragment(svg)).filter(isElement)[0];
+}
+
+const symbolRe = /<symbol([^]*?id=(['"])(.+?)\2[^]*?<\/)symbol>/g;
```

Biome (1.9.4)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)